web/beacon-app/src/features/projects/components/ProjectList.tsx

```diff
@@ -27,10 +29,16 @@ function ProjectList() {
 
   return (
     <>
-      <Heading as="h1" className="mb-4 mt-6 text-lg font-semibold">
-        Projects
-      </Heading>
+      <div className="flex space-x-2 space-y-2">
+        <Heading as="h1" className="mb-4 mt-6 text-lg font-semibold">
+          Projects
+        </Heading>
+        <ProjectTooltip />
+      </div>
       <div className="flex justify-between rounded-lg bg-[#F7F9FB] px-3 py-2">
+        <div className="mt-2">
+          <RefreshIcon />
```

Comment:
Maybe the refresh icon could be wrapped within a button? i'm wondering if it's not better to implement directly the logic ?

Reply:
Thanks. I kept forgetting buttons yesterday for some reason.